dfaker.py

```diff
@@ -91,426 +82,67 @@ def parse(args, params):
-        next_time += basal_entry["duration"] / 1000
-        dfaker.append(basal_entry)
-        
-def settings(start_time, params):
-    settings = {}
-    time_in_seconds = int(start_time.strftime('%s'))
-    settings = add_common_fields('settings', settings, time_in_seconds, params)
-    settings["activeSchedule"] = "standard"
-    settings["basalSchedules"] =  {"standard": [{"rate": 0.9, "start": 0},
-                                                {"rate": 0.6, "start": 3600000},
-                                                {"rate": 0.65, "start": 10800000},
-                                                {"rate": 0.8, "start": 14400000},
-                                                {"rate": 0.85, "start": 18000000},
-                                                {"rate": 0.8, "start": 28800000},
-                                                {"rate": 0.75, "start": 32400000},
-                                                {"rate": 0.8, "start": 54000000},
-                                                {"rate": 0.85, "start": 61200000}]}
-    bgTarget_low = random.randrange(80, 120, 10)
-    bgTarget_high = random.randrange(bgTarget_low, 140, 10)
-    settings["bgTarget"] = [{"high": convert_to_mmol(bgTarget_high), 
-                             "low": convert_to_mmol(bgTarget_low), 
-                             "start": 0}]
-    settings["carbRatio"] = [ {"amount": random.randint(9, 15), "start": 0}]
-    settings["insulinSensitivity"] = [{"amount": convert_to_mmol(50), "start": 0}]
-    settings["units"] = { "bg": "mg/dL","carb": "grams"}
-    dfaker.append(settings)
-
-dfaker = [] 
-solution = cbg_equation.stitch_func(params['num_days'])
-
-d = params['datetime']
-start_time = datetime(d.year, d.month, d.day, 
-                    d.hour, d.minute, tzinfo=timezone(params['zone']))
-
-cbg_gluc, cbg_time, smbg_gluc, smbg_time = apply_loess(params, solution)
-cbg_timesteps = make_timesteps(start_time, cbg_time)
-smbg_timesteps = make_timesteps(start_time, smbg_time)
-
-b_carbs, b_carb_timesteps, w_carbs, w_carb_timesteps, w_gluc = generate_boluses(solution, start_time)
-
-#add settings to dfaker
-settings(start_time, params)
-#add basal to dfaker
-scheduled_basal(start_time, params)
-#add bolus values to dfaker
-bolus(b_carbs, b_carb_timesteps, params)
-#add wizard events to dfaker
-wizard(w_gluc, w_carbs, w_carb_timesteps, params)
-#add cbg values to dfaker
-cbg(cbg_gluc, cbg_timesteps, params)
-#add smbg values to dfaker
-smbg(smbg_gluc, smbg_timesteps, params)
+        json.dump(dfaker, fp=file_object, sort_keys=True, indent=4) 
+    file_object.close()
 
-#write to json file
-file_object = open(params['file'], mode='w')
-if params['minify']: #for most compact file: separators=(',', ':')
-    json.dump(dfaker, fp=file_object, sort_keys=True, separators=(',', ':'))
-else:
-    json.dump(dfaker, fp=file_object, sort_keys=True, indent=4) 
-file_object.close()
+if __name__ == '__main__':
+    main()
 
-sys.exit(0)
```

Comment:
This is probably a dreg from the reorganization and adding main. The `sys.exit(0)` should *not* be outside `main()` - it means if you try to import dfaker.py as a module, it exits immediately.